packages/gatsby/src/utils/api-node-docs.js

```diff
@@ -203,3 +203,9 @@ exports.onPostBuild = true
  * See gatsby-transformer-remark and gatsby-source-contentful for examples.
  */
 exports.onPreExtractQueries = true
+
+/**
+ * Run after the schema has been created but before it's dispatched allowing you
+ * to do cool enhancements e.g. union / interface stuff
+ */
+exports.enhanceSchema = true
```

Comment:
Per the API specification, this should be `setSchemaFields` https://www.gatsbyjs.org/docs/api-specification/#operators

Reply:
Actually, let's make this `createSchemaFields`. There's no reason to allow people to override the default fields so we should validate against that.